dist/development/styles/strigo-widget.css

```
@@ -44,20 +45,49 @@
   border-right: 2px #adafdc solid;
 }
 #strigo-widget .strigo-collapse-div {
-  width: 6px;
+  width: 4px;
   height: 100%;
   background: #3d408f;
-  border-left: 2px #adafdc solid;
-  box-shadow: 0px 1px 4px 1px rgba(0, 0, 0, 0.3);
-  cursor: pointer;
+  border: none;
 }
 #strigo-widget .strigo-collapse-div:hover {
   background: #696cbf;
 }
-#strigo-widget .strigo-collapse-div:hover {
-  background: #696cbf;
+#strigo-widget .strigo-collapse-div.slide-in {
+  display: none;
+}
+.strigo-academy-hat {
```

WDYT about using ids here, instead of classes?

```suggestion
#strigo-academy-hat {
```